sections/05_treat.md

```diff
@@ -162,45 +162,193 @@ feature selection and construction.*
+ranking problem.  In reality, it does not fit neatly into any of those
+categories.  An ideal algorithm will rank a sufficient number of active
+compounds before the inactives, but the rankings of actives relative to other
+actives and inactives to other inactives is less important
+[@tag:Swamidass2009_irv]. `TODO: can improve this first attempt at an intro by
+reviewing more existing literature on the topic` `TODO: check which other
+existing reviews should be referenced`
+
+We primarily focus on ligand-based approaches that train on chemicals' features
+without requiring knowledge of the target, as opposed to alternative strategies
+that use target features such as the protein structure `TODO: add examples`.
+Neural networks have a long history in this domain [@tag:Baskin2015_drug_disc]
+`TODO: can add additional references besides this review`, and the 2012 Merck
+Molecular Activity Challenge on Kaggle `TODO: need URL?` generated substantial
+excitement about the potential for high-parameter deep learning approaches.  The
+winning submission was an ensemble that included a multitask multilayer
+perceptron network [@tag:Dahl2014_multi_qsar], and the Merck sponsors noted
+drastic improvements over a random forest (RF) baseline, remarking "we have
+seldom seen any method in the past 10 years that could consistently outperform
+RF by such a margin" [@tag:Ma2015_qsar_merck].  Subsequent work explored the
+effects of jointly modeling far more targets than the Merck challenge
+[@tag:Unterthiner2014_screening @tag:Ramsundar2015_multitask_drug], with
+[@tag:Ramsundar2015_multitask_drug] showing that the benefits of multitask
+networks had not yet saturated even with 259 targets.  Although a deep learning
+approach, DeepTox, was also the overall winner of another competition, the
+Toxicology in the 21st Century (Tox21) Data Challenge, it did not dominate
+alternative methods as thoroughly as in other domains.  DeepTox was the top
+performer on only 9 of 15 targets, and in some cases there was little separation
+between DeepTox and the second best method on those 9 tasks.  A reliance on AUC
+ROC `TODO: define here?` for the evaluation (see Discussion) further hinders the
+ability to declare Tox21 as an outright success for deep learning.
+
+In retrospect, the nuanced Tox21 performance may be more reflective of the
+practical challenges encountered in ligand-based chemical screening than the
+extreme enthusiasm generated by the Merck competition.  A study of 22 absorption
+distribution, metabolism, excretion, and toxicity (ADMET) tasks demonstrated
+that there are limitations to multitask transfer learning that are in part a
+consequence of the degree to which tasks are related [@tag:Kearnes2016_admet].
+Some of the ADMET datasets showed far superior performance in multitask models
+of only the 22 ADMET tasks relative to more expansive multitask networks that
+included over 500 less-similar tasks.  `TODO: also has a good discussion of
```

Comment:
Sounds good.  I'm leaving the TODO here for now to simplify the merge but can write about this in Discussion.